### PR TITLE
add button and card components

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: components.css [hunk 1]
```diff
@@ -261,4 +261,242 @@
...
+    opacity: 0.5;
+}
+
+.card-content-container{
+    display: flex;
+    flex-direction: column;
+}
+
+.card-main-content{
+    display: flex;
```
Comment (mariyasada) [inline components.css:493]: Using display:flex two times

File: styles.css [hunk 2]
```diff
@@ -460,4 +461,248 @@ iframe{
...
+
+.card-content-container{
+    display: flex;
+    flex-direction: column;
+}
+
+.card-main-content{
+    display: flex;
+    gap: 1rem;
+    display: flex;
```
Comment (mariyasada) [inline styles.css]: Here also

**Review by mariyasada — APPROVED** [review]
Look at my comments

File: styles.css (hunk 2 above)
Comment (ganesh-kanchi) [reply styles.css]: fixed them. thanks for reviewing.

File: components.css [hunk 3]
```diff
@@ -261,4 +261,242 @@
...
+    align-content: center;
+    margin: auto;
+    padding: 0;
+    border-radius: 2%;
+    box-shadow: 3px 3px 15px var(--black);
+    background-color: var(--white);
+    position: relative;
+}
+
+/* .image-container{
```
Comment (RakshithBhat03) [inline components.css]: You can remove comments

File: components.css [hunk 4]
```diff
@@ -261,4 +261,242 @@
...
     border: none;
     border-radius: 50%;
+}
+
+.button{
+    padding: 0.75rem 1rem;
+    margin: 1rem;
+    border-radius: 0.5rem;
+    border: none;
+    color: white;
```
Comment (RakshithBhat03) [inline components.css]: You can use a variable here.

**Review by RakshithBhat03 — APPROVED** [review]
Suggested some changes. Everything else looks good.

File: components.css (hunk 4 above)
Comment (ganesh-kanchi) [reply components.css]: made the changes, thanks for reviewing.

**Pull request merged**